app/views/comments/_comments.html.erb

```diff
@@ -34,11 +34,13 @@
       <td><%= distance_of_time_in_words(comment.created_at, Time.current, { include_seconds: false, scope: 'datetime.time_ago_in_words' }) %></td>
       <td>
         <% if comment.status == 4 && current_user &. can_moderate? %>
+              Status:<%= comment.status %>
               <a class="btn btn-outline-secondary btn-sm" href="/admin/publish_comment/<%= comment.id %>"><%= translation('dashboard.moderate.approve') %></a>
               <a class="btn btn-outline-secondary btn-sm" href="/admin/mark_comment_spam/<%= comment.id %>"><%= translation('dashboard.moderate.spam') %></a>
         <% elsif current_user &. can_moderate? %>
+            Status:<%= comment.status %>
```

Comment:
Could you add a space here between status: and the number.
Also for the same part that exists above. Thanks!

Reply:
@Uzay-G Sure, i will add this